### ISSUE TITLE
Prestashop Metrics issue with icons

### ISSUE THREAD

**Issue by MRemeikis** [PrestaShopCorp/ps_mbo#779]
Hello! I upgraded my prestashop from 7.4 version to 8.1.2 and after the upgrade all the icons in the back office seem to be gone except for when i am on the main page of the back office (i am including the screenshots)

![image](https://github.com/user-attachments/assets/17bb375e-d2e9-4752-9ccd-60102fbf0ea9)

![image](https://github.com/user-attachments/assets/98e6a8bc-2cb5-467c-b94e-5c0cb89a69d5)

Interesting thing, if i turn off the module, then the icons are missing in the main page of the back office too.

Also i've been reading about an error with accessing orders and twig issue. When i have a ets_ordermanager module on , i get this error and some people on some pages wrote that this is ps_mbo that is causing this issue

![image](https://github.com/user-attachments/assets/53fa921e-c1c4-4826-afb2-84ca755f3e1b)

When i change the code to the path to be @Prestashop instead of Prestashop bundle like the error shows i get an internal server error after a long loadtime and nothing in the logs of var/logs that could show me what could be the error

**Comment by intraordinaire** [comment]
Hello @MRemeikis ,

Thank you for your feedback.

Not sure this is directly linked to ps_metrics.

Have you tried to : 
 - Rebuild assets ?
 - Empty PrestaShop cache (delete everything under `var/cache`)
 - Clear/empty browser cache ?

For the `ets_ordermanager` and order listing bug, I'm curious, never heard about it.
I just tried to install it, and everything seems to work well.
Here again, maybe, clearing the PrestaShop cache may be the solution.

If not, let me know, I'll try to help you further.

Thanks

**Comment by MRemeikis** [comment]
i did try to empty all cache, browser and in the server under var/cache but it didn't help. Mind me asking where i could find the function to rebuild assets? I might have done it, but since at the moment, my prestashop is in different language, i might need your guidance where to find it :D


As for ets_ordermanager, it is a module by prestahero that was bought a long time ago by my company, and at the moment i am the newest member of the website maintenance and the only one at it. As far as I noticed when I upgraded at first to 8.1.7 it had the same errors and crashes, then I rollbacked, and upgraded to the version that module at it's current version could support, which was 8.1.2, and it has many errors (i can link another git ticket i opnened on prestashop for more info)

https://github.com/PrestaShop/PrestaShop/issues/37175

**Comment by intraordinaire** [comment]
@MRemeikis to rebuild the assets, you can do as mentioned [here](https://devdocs.prestashop-project.org/8/development/compile-assets/). 
TLDR : you need an ssh access to the machine, node14 installed on it. Go to the root folder, and `make admin` to rebuild assets for back office. If not working properly, follow the more detailed way in the doc given.

For your others problems I'm quite curious, I'm working on a PR #781 to simplify some things, and I think the module ps_mbo will be less heavy to run after that, but not sure this can help you.
Do you have access to the logs (under `var/logs` and `dev.log` or `prod.log`) and check if you have others critical issues before/around the one displayed ? (you can copy some of the lines, just be sure to remove sensitive info before).

**Comment by MRemeikis** [comment]
well i have many issues with modules and etc after upgrading to a prestashop 8.1.2 , like i cannot even use a checkout at the moment from the front store, sais issues with controllers. I think i will just log this issue to the other git. As for mbo i think i mistook the modules and mbo module != ps_metrics lol , my bad if i logged it into the wrong place

**Comment by intraordinaire** [comment]
@MRemeikis don't worry about that, we're happy to help if we can.
If you have logs about the problem on `ps_metrics`, you can still post them here if you want, and we'll try to see with the dedicated team to check.
Hope you'll find solutions to your bugs !

Feel free to reopen the issue, or open a new one if you encounter any trouble.
